rust/crd/src/supersetdb.rs

```diff
@@ -55,7 +55,7 @@ impl SupersetDB {
             // when the cluster is created again.
             metadata: ObjectMetaBuilder::new()
                 .name_and_namespace(superset)
-                .with_recommended_labels(superset, APP_NAME, version, "", "") // TODO fill in missing fields
+                .with_recommended_labels(superset, APP_NAME, version, "", "", "") // TODO fill in missing fields
```

Comment:
Resolve the TODO or explain why this is empty?

Reply:
Some labels should not be set because SupersetDB is not owned by the cluster as mentioned in the comment. So instead of providing empty values, only the necessary labels are added explicitly.